go/database/mpt/live_trie_test.go

```diff
@@ -893,6 +893,57 @@ func TestLiveTrie_VerificationOfLiveTrieWithCorruptedFileFails(t *testing.T) {
 	}
 }
 
+func TestLiveTrie_HasEmptyStorage(t *testing.T) {
+	for _, config := range allMptConfigs {
+		t.Run(config.Name, func(t *testing.T) {
```

Comment:
it should also test when the storage emptied by other ways: ClearStorage, set Value to zero